.babelrc

```diff
@@ -21,6 +20,9 @@
     "@babel/plugin-transform-react-jsx",
     "@babel/plugin-transform-react-constant-elements",
     "@babel/plugin-transform-runtime",
-    "@babel/plugin-proposal-class-properties"
-  ]
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-syntax-dynamic-import",
+    "react-hot-loader/babel"
+  ],
+  "sourceType": "unambiguous"
```

Comment:
`@babel/plugin-syntax-dynamic-import` is needed to allow the [dynamic import](https://webpack.js.org/guides/code-splitting/#dynamic-imports) syntax used for code splitting.

`react-hot-loader/babel` is https://github.com/gaearon/react-hot-loader - basically it lets us edit a component file and have the changes automagically reload in the browser. I only enabled it for the external links editor for now, just to test it out. (You'll need to be running `./script/compile_resources.sh --watch`.)

`"sourceType": "unambiguous"` is explained in e90bd3fa297b41292548b2bc1dd7dad69a2398bc.

Reply:
I've removed `react-hot-loader` from this PR to reduce the amount of things needing testing. (`--watch` mode still exists, but you'll have to reload the page manually.) We can always add it later.